nixos/tests/docker-tools.nix

```diff
@@ -16,6 +16,17 @@ import ./make-test.nix ({ pkgs, ... }: {
       };
   };
 
+  runAsRootExtraCommands = pkgs.dockerTools.buildImage {
```

Comment:
Arf, sorry, I haven't seen this image was in this file:/ Could you either move it to `docker/examples.nix` or in a 
```nix
let runAsRootExtraCommands =... 
in import ./make-test.nix....
```